Hackney.Shared.Processes/Hackney.Shared.Processes.csproj

```diff
@@ -3,7 +3,7 @@
     <TargetFramework>netstandard2.1</TargetFramework>
     <PackageId>Hackney.Shared.Processes</PackageId>
     <RepositoryUrl>https://github.com/LBHackney-IT/processes-shared</RepositoryUrl>
-    <Version>0.0.4</Version>
+    <Version>0.0.6</Version>
```

Comment:
We don't this in the csproj, we have automatic versioning that happens as part of the pipeline

Reply:
Amended.